test/test-run-trace.js

```diff
@@ -6,15 +6,16 @@
 'use strict';
 
 var debug = require('./debug');
-var helper = require('./helper');
-var http = require('http');
-var run = helper.runWithControlChannel;
+var run = require('./run-with-ctl-channel');
 var tap = require('tap');
 
 var skipIfNoLicense = process.env.STRONGLOOP_LICENSE
                     ? false
```

Comment:
appmetrics doesn't have licensing, so metrics doesn't need it, but strong-trace still does: https://github.com/strongloop/strong-trace/blob/master/strong-trace.js#L20